test/test.js

```diff
@@ -42,6 +42,7 @@
   var phantom = root.phantom,
       amd = root.define && define.amd,
       argv = root.process && process.argv,
+      jsArrayFrom = Array.from,
```

Comment:
I still need to add unit tests for `_.toArray` and the like (more for this evening).